### ISSUE TITLE
Image blocks don’t have proper alt="" text set

### ISSUE THREAD

**Issue by TheoChevalier** [mozilla/foundation.mozilla.org#7520]
All images should use the dedicated `AltText` field for their `alt=""` attribute, and use the translated version on translated pages.
Currently, image blocks are using the `Title` field, which is not translated and sometimes consists of just the filename:

<img width="828" alt="Capture d’écran 2021-09-28 à 15 40 48" src="https://user-images.githubusercontent.com/1294206/135098390-9a0c0ac2-d0db-401e-bd7a-ad1a286d171f.png">

Example from https://foundation.mozilla.org/fr/what-we-do/, all images are affected

**Comment by benhohner** [comment]
Proper behaviour is alt tag should be empty if there is no description or title for image. @richbrennan can you please verify if this is being fixed in Wagtail core or not?

**Comment by richbrennan** [comment]
Hey @benhohner 

So by default Wagtail uses the title for the alt text: https://github.com/wagtail/wagtail/blob/main/wagtail/images/models.py#L375

If no title is entered when the image is uploaded then the title is populated with the filename, hence why filenames appear in alt tags.

It looks like there [aren't any plans](https://github.com/wagtail/rfcs/blob/image-models/text/044-image-models.md#update-the-project-template-to-use-custom-image-rendition-and-document-models) to add an alt field to the image model in Wagtail core, so the solution would be to use a custom image model and add it in the project.

I'll investigate whether image titles are being translated.

**Comment by richbrennan** [comment]
Further to the above, the out-of-the-box Wagtail image model doesn't support translations as it doesn't inherit from `TranslatableMixin`.

I've asked Karl whether a custom image model could inherit from `TranslatableMixin` so the text fields could be translated. I'll update again once I get a reply.

**Comment by richbrennan** [comment]
With regard to image _blocks_, these are StreamField StructBlocks with an `AltText` field, however [at least one of them](https://github.com/mozilla/foundation.mozilla.org/blob/main/network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/image_block.html#L6) isn't using `AltText` for its alt attribute, so the fix would be to update the relevant templates to use the correct value.

**Comment by richbrennan** [comment]
Pull request for review: https://github.com/mozilla/foundation.mozilla.org/pull/7839
